src/common/buffer.cc

```diff
@@ -386,9 +391,16 @@ using namespace ceph;
 #ifdef DARWIN
       data = (char *) valloc(len);
 #else
-      int r = ::posix_memalign((void**)(void*)&data, align, len);
-      if (r)
-	throw bad_alloc();
+      if (len == 2_M) {
```

Comment:
This is just a dirty hack to verify the benefit `BlueFS::FileWrite` can get from jumbo pages.